doc-content/enterprise-only/cases/case-management-creating-order-placed-milestone-proc.adoc

```diff
@@ -0,0 +1,37 @@
+[id='case-management-create-order-placed-milestone-proc']
+= Creating the Order placed milestone
+This milestone is reached when the `ordered` variable has been given a response.
```

Comment:
I would specify here, that "ordered" variable is part of "Place order" sub-process.